astropy/table/tests/test_table.py

```diff
@@ -2150,10 +2150,10 @@ def test_masking(self, use_nullable_int):
             # No warning with the default use_nullable_int=True
             d = t.to_pandas(use_nullable_int=use_nullable_int)
         else:
-            import pandas
+            import pandas as pd
             from packaging.version import Version
 
-            PANDAS_LT_2_0 = Version(pandas.__version__) < Version("2.0")
+            PANDAS_LT_2_0 = Version(pd.__version__) < Version("2.0")
```

Comment:
Would it make sense to use `minversion("pandas", "2.0")` from `astropy.utils.introspection` here as well to remove the `import pandas` altogether?

Reply:
Yes, then we wouldn't need to import `Version` either.